src/constants/request.ts

```diff
@@ -52,6 +52,7 @@ export const URLs = {
   },
   subjects: {
     get: '/subjects',
+    getByCategoryId: '/categories/:categoryId/subjects',
```

Comment:
You don't need to add named parameters when there is only one of them
```suggestion
    getByCategoryId: '/categories/:id/subjects',
```

Reply:
Fixed.